extensions/amp-analytics/0.1/test/test-events.js

```diff
@@ -157,6 +158,192 @@ describes.realWin('Events', {amp: 1}, env => {
     });
   });
 
+  describe('ScrollEventTracker', () => {
+
+    let tracker;
+    let fakeViewport;
+    let getFakeViewportChangedEvent;
+    const defaultScrollConfig = {
+      'on': 'scroll',
+      'scrollSpec': {
+        'verticalBoundaries': [0, 100],
+        'horizontalBoundaries': [0, 100],
+      },
+    };
+    let scrollManager;
+
+    beforeEach(() => {
+      tracker = root.getTracker('scroll', ScrollEventTracker);
+      fakeViewport = {
+        'getSize': sandbox.stub().returns(
+            {top: 0, left: 0, height: 200, width: 200}),
+        'getScrollTop': sandbox.stub().returns(0),
+        'getScrollLeft': sandbox.stub().returns(0),
+        'getScrollHeight': sandbox.stub().returns(500),
+        'getScrollWidth': sandbox.stub().returns(500),
+        'onChanged': sandbox.stub(),
+      };
+      scrollManager = tracker.root_.getScrollManager();
+      scrollManager.viewport_ = fakeViewport;
+
+      getFakeViewportChangedEvent = () => {
+        const size = fakeViewport.getSize();
+        return {
+          top: fakeViewport.getScrollTop(),
```

Comment:
please update the ScrollEvent here

Reply:
Discussed offline, Viewport Events -> Scroll Events by the Scroll manager